helpers/reminderHelper.js

```diff
@@ -0,0 +1,26 @@
+const { getGuild, saveGuild } = require('./guildData')
+const { discordTime } = require('./DateTimeHelper')
+const { foemp } = require('./foemp')
+
+async function setReminder (interaction) {
+  const guild = await getGuild(interaction.guildId)
+  const currentTime = discordTime().toEpochSecond().toString()
+  const reminderMinutes = interaction.fields.getTextInputValue('reminderTime')
+  if (isNaN(reminderMinutes)) {
+    return await interaction.reply({ content: `Je hebt het aantal minuten ingesteld (${reminderMinutes}) in een formaat dat ik niet begrijp. Probeer een integer (bv. 10), ${foemp()}!`, ephemeral: true })
+  }
+  const timeToBeReminded = discordTime().plusMinutes(interaction.fields.getTextInputValue('reminderTime')).toEpochSecond()
```

Comment:
You already calculated `reminderMinutes` so you can reference it here instead of doint that again